client/src/api/results.js

```diff
@@ -0,0 +1,79 @@
+export async function fetchResults() {
+  // TODO: Connect API here later
+  /*
+  const response = await fetch("localhost:3001/app/offers", {
+    header: {
+      //TODO:
+    },
+  });*/
+
+  //const result = await response.json();
```

Comment:
I would suggest removing comments like this. Better create issues on GitHub or use different solutions like code extensions to handle todos. 
The main reason is, that this todo leads to more confusing and there is a chance, that you forget to remove them. 

`localhost:3001` will lead to other issues. Take a look at [proxy](https://create-react-app.dev/docs/proxying-api-requests-in-development).

Reply:
Understandable. Thanks for your advice, will change!

Reply:
done.